src/components/Section.js

```diff
@@ -1,17 +1,15 @@
 export default class Section {
   constructor({ items, renderer }, containerSelector) {
-    this._renderer = renderer;
     this._items = items;
-    this._containerSelector = document.querySelector(`${containerSelector}`);
+    this._renderer = renderer;
+    this._containerSelector = document.querySelector(containerSelector);
   }
   renderItems() {
-    this._items.forEach((item) => {
-      this._renderer(item);
-    });
+    this._items.forEach(this._renderer);
   }
 
-  addItem(data) {
-    this._containerSelector.prepend(data);
+  addItem(cardEl) {

```

Comment:
Needs correction: `Section` is a general class. It can render any kind of items, not only cards. Please, rename the argument to something like `item`